src/go/rpk/pkg/cli/cmd/generate/prometheus.go

```diff
@@ -85,10 +85,7 @@ especify an arbitrary config file.`,
 		&nodeAddrs,
 		"node-addrs",
 		[]string{},
-		fmt.Sprintf(`
-A comma-delimited list of the addresses (<host:port>) of all the redpanda nodes
-in a cluster. The port must be the one configured for the nodes' admin API
-(%d by default)`,
+		fmt.Sprintf(`A comma-delimited list of the addresses (<host>:<port>) of all the redpanda nodes in a cluster. The port must be the one configured for the nodes' admin API (%d by default)`,
```

Comment:
I'm ok with this change (and may shorten the wording a bit later), but why merge in this PR?

Reply:
Since the script that I wrote try to read each line, this extra was giving me some trouble. So it's a good idea to standardize it now.